code/datums/components/embedded.dm

```diff
@@ -286,8 +293,11 @@
 			to_chat(victim, span_danger("[user] fails to pluck [weapon] from your [limb.plaintext_zone]."))
 		return
 
-	to_chat(user, span_notice("You successfully pluck [weapon] from [victim]'s [limb.plaintext_zone]."))
-	to_chat(victim, span_notice("[user] plucks [weapon] from your [limb.plaintext_zone]."))
+	to_chat(user, span_notice("You [tweezer_safe ? "safely" : "unsafely, but"] successfully pluck [weapon] from [victim]'s [limb.plaintext_zone]."))
```

Comment:
```suggestion
	to_chat(user, span_notice("You [tweezer_safe ? "" : "unsafely, but"] successfully pluck [weapon] from [victim]'s [limb.plaintext_zone]."))
```
I don't think the `safely` adverb is needed, plus you've forgot to add a conjuction between it and `successfully` anyway.